### ISSUE TITLE
[Enhancement] - Adding leaves & objects

### ISSUE THREAD

**Issue by schme16** [mattatz/unity-generative-tree#3]
First off thank you so much for the work you've done making this asset, it's been exactly what I needed (after building a URP shader for the growth, haha)!

I'm now trying to retrofit in the ability to place leaves and other objects (apples in my case), and though I'm muddling through ok by trying to read through your code and determine locations for things to spawn, I'm just not skilled enough with mesh manipulation to get it right.

TL;DR: I'd love to see this enhanced with the option to provide a leaf, and fruit, prefab, and have the tree calculate the appropriate locations to spawn them in.

Thanks
